localMatchApp/sources/Login.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Login.h"
+
+using namespace std;
+
+Login::Login ( ) {
+  // Connect to database.
+}
+
+Login::~Login ( ) {
+  // Disconnect to database.
```

Comment:
Add a TODO comment for this pending task.

Reply:
Done.